test/extended/operators/olm.go

```diff
@@ -87,4 +88,22 @@ var _ = g.Describe("[Feature:Platform] OLM should", func() {
 			o.Expect(msg).To(o.Equal("IfNotPresent"))
 		}
 	})
+
+	// OCP-21082 - Implement packages API server and list packagemanifest info with namespace not NULL
+	// author: bandrade@redhat.com
+	g.It("Implement packages API server and list packagemanifest info with namespace not NULL", func() {
+		msg, err := oc.AsAdmin().WithoutNamespace().Run("get").Args("packagemanifest", "--all-namespaces", "--no-headers").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		var packageserverLine string
+		for _, line := range strings.Split(msg, "\n") {
+			if strings.Contains(line, "etcd") {
+				packageserverLine = line
+				break
+			}
+		}
+		if !strings.Contains(packageserverLine, "openshift-marketplace") {
```

Comment:
There is a strong chance that this namespace will change at some point (given that long term we want to deprecate the marketplace apis entirely). Can we also just ensure that this is a non-nil string?

Reply:
Marking as resolved, please check https://github.com/openshift/origin/pull/23772#discussion_r354428150